operators/src/engine/result_descriptor.rs

```diff
@@ -120,6 +120,34 @@ impl ResultDescriptor for VectorResultDescriptor {
     }
 }
 
+/// A `ResultDescriptor` for plot queries
+#[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Serialize, Deserialize)]
+pub struct PlotResultDescriptor {}
+
+impl ResultDescriptor for PlotResultDescriptor {
+    type DataType = (); // TODO: maybe distinguish between image, interactive plot, etc.
```

Comment:
The basic distinction would be between backend and frontend rendered plots. And the type also has to correspond to the `TypedPlotQueryProcessor`. But how is that relevant for the client? Why does he need to know how the plot is rendered? It's not like for other layers where we need the info because they are used as input for operators.

Reply:
Vega rendering, for instance, is some kind of special JSON file. The client would need to render it differently. But I guess this will be more clear if we try to put it together with the front end. For now, it's empty.